### PR TITLE
Render correct csv table data; populate c3 line chart with csv table data

### PR DESCRIPTION
**Pull request by cmperez**
Sorry about the c3.css file (makes up 180+ lines of this commit)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,8 @@
     "bluebird": "^2.5.2",
     "body-parser": "^1.10.0",
     "busboy": "^0.2.9",
+    "c3": "^0.4.7",
+    "d3": "^3.5.3",
```
Comment (dduugg) [inline package.json]: you're adding d3 but not requiring it anywhere? i assume it's a dependency of c3 then... but not in c3's package.json?

File: client/js/components/data_page.jsx [hunk 2]
```diff
@@ -81,20 +77,25 @@ var Table = React.createClass({
...
+      });
       return (
         <div className="data-page">
           <aside className="tools">
             <input className="data-search" placeholder="Search schemas" type="text"/>
           </aside>
           <section className="viz-and-table-views">
-            <div className="visualization-view">Chart goes here</div>
+            <div className="visualization-block">
+              <div className="visualization-label">{this.props.schemaName}</div>
```
Comment (cmperez) [inline client/js/components/data_page.jsx]: totes fine to have plain text in a div

File: package.json (hunk 1 above)
Comment (cmperez) [inline package.json]: I suspect it's unnecessary. I'll try deleting.

File: client/js/components/data_page.jsx (hunk 2 above)
Comment (zindlerb) [inline client/js/components/data_page.jsx]: :cool:

**Comment by dduugg** [comment]
:+1:

**Comment by zindlerb** [comment]
:+1:

**Pull request merged**